tests/Exceptions/ValidationExceptionTest.php

```diff
@@ -98,4 +111,3 @@ public function providerForFormat()
         );
     }
 }
-
```

Comment:
Yeah, GitHub doesn't display the last line but there is an special char to indicate when there is no new line at end. That's why the common standard is to have one new line at the end of file, so, there is no need on having more than one, but just once.